ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.13.yaml

```diff
@@ -489,6 +489,31 @@ tests:
       BRANCH: "4.13"
       TEST_TYPE: conformance-serial
     workflow: openshift-e2e-powervs-ipi
+- as: ocp-e2e-azure-ovn-heterogeneous
```

Comment:
You missed a make update before committing

Reply:
Ack @aleskandro - Also, I will update for 4.14 release as well.